### PR TITLE
19 admin logs in

### PR DESCRIPTION
**Pull request by JoelLindow**
Related Issue: #9 

Summary: 
- added admin controller
- added namespacing in routes
- added admin/admin views to reflect namespacing

Author(s): Joel and Randy

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ def create
     if user && user.authenticate(params[:session][:password])
       session[:user_id] = user.id
       flash[:success] = "Logged in as #{user.username}"
-      redirect_to dashboard_path
+      redirect_to dashboard_path if current_user.default?
+      redirect_to admin_dashboard_path if current_user.admin?
```
Comment (randy-darsh) [inline app/controllers/sessions_controller.rb:12]: this was a good catch

File: config/routes.rb [hunk 2]
```diff
@@ -12,6 +12,10 @@
 
   resources :orders, only: [:index, :show]
 
+  namespace :admin do
```
Comment (limsammy) [inline config/routes.rb:15]: Nice use of namespace, still need to get used to that.

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (limsammy) [inline app/controllers/sessions_controller.rb:12]: Do we need to create the admin? method?

**Review by limsammy — APPROVED** [review]
Looks good, merging

**Pull request merged**